app/models/account/provider_methods.rb

```diff
@@ -338,13 +338,11 @@ def create_service(attrs)
   protected
 
   def create_first_service
-    return if @signup_mode
+    return if @signup_mode || has_bought_cinstance?
```

Comment:
That is not the same, in this case you will never run the ServiceCreator

Reply:
I know, that's why I left it as "WIP". sorry that I only spoke about it in private to another person 😞